metis-core/metis-core-rest/src/main/java/eu/europeana/metis/core/rest/config/ElasticAPMConfiguration.java

```diff
@@ -0,0 +1,49 @@
+package eu.europeana.metis.core.rest.config;
+
+import co.elastic.apm.attach.ElasticApmAttacher;
+import java.util.HashMap;
+import java.util.Map;
+import javax.annotation.PostConstruct;
+import org.apache.commons.lang3.SerializationUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * Elastic apm configuration
+ */
+@Configuration
+@ConfigurationProperties(prefix = "elastic")
+@ConditionalOnProperty(value = "elastic.apm.enabled", havingValue = "true")
```

Comment:
nice, I think we can use this on the sandbox as well :100:

Reply:
Thanks! I find this a great way to opt-in and opt-out.